libraries/ESP32/examples/MacAddress/GetMacAddress.ino

```diff
@@ -0,0 +1,85 @@
+/*

```

Comment:
For Arduino IDE users sketch name and directory name should be identical IIRC.

Reply:
Of course!  The path is now: 'libraries/ESP32/examples/MacAddress/GetMacAddress/GetMacAddress.ino'

Thank you!